SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -152,4 +154,6 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	TMap<Worker_ComponentId, TSharedRef<FClassInfo>> ComponentToClassInfoMap;
 	TMap<Worker_ComponentId, uint32> ComponentToOffsetMap;
 	TMap<Worker_ComponentId, ESchemaComponentType> ComponentToCategoryMap;
+
+	TOptional<bool> bHandoverActive;
```

Comment:
I never know whether these things are guaranteed to be correctly initialized

Reply:
It'll be initialised to `unset`, which is what we want until we set it.